index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] From the project requirement, you are supposed to add a descriptive class, and style it, but It seemed that you've not added the class to your `h1` tag.  Please add your the class name as required.

Reply:
- hello @ishimwezachee  thank you 👍 for your helpful feedback.I will do my best to implement changes